src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -87,16 +82,32 @@ angular
             y: -5
         }]
 
-    thresholdsMarkers: (options = {})->
-      return {} if _.isEmpty(options.thresholds)
-      lines = []
+    addThresholds: (options = @options)->
```

Comment:
Just out of curiosity: did you try to keep the plotLine, adding a zone with color = transparent before today?
I'm just wondering if we can avoid the fake serie and adding all this code?..